### ISSUE TITLE
[DSCForLinux] install fails

### ISSUE THREAD

**Issue by Madhura2016** [Azure/azure-linux-extensions#203]
Hi,
I am following the following code from https://azure.microsoft.com/en-us/blog/introduce-azure-dscforlinux-extension/ to get Azure DSC Automation Extension for Linux:

```
$rgName = '<resource-group-name>'
$vmName = '<vm-name>'
$location = '<location>'

$extensionName = 'DSCForLinux'
$publisher = 'Microsoft.OSTCExtensions'
$version = '<version>'

$privateConfig = '{
  "StorageAccountName": "<storage-account-name>",
  "StorageAccountKey": "<storage-account-key>",
  "ContainerName": "<container-name>",
  "MofFileName": "<mof-file-name>"
}'

$publicConfig = '{
  "Mode": "Push"
}'
Set-AzureRmVMExtension -ResourceGroupName $rgName -VMName $vmName -Location $location `
  -Name $extensionName -Publisher $publisher -ExtensionType $extensionName `
  -TypeHandlerVersion $version -SettingString $publicConfig -ProtectedSettingString $privateConfig
```

But I get and error as follows:

```
Set-AzureRmVMExtension : Long running operation failed with status 'Failed'.
StartTime: 7/14/2016 2:22:37 PM
EndTime: 7/14/2016 2:23:44 PM
OperationID: 24a97b83-3714-4ffa-b7cc-b8da0d4ae211
Status: Failed
ErrorCode: VMExtensionProvisioningError
ErrorMessage: VM has reported a failure when processing extension 'DSCForLinux'. Error message: "Install Failed.".
At line:19 char:1
+ Set-AzureRmVMExtension -ResourceGroupName $rgName -VMName $vmName -Lo ...
+ ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : CloseError: (:) [Set-AzureRmVMExtension], ComputeCloudException
    + FullyQualifiedErrorId : Microsoft.Azure.Commands.Compute.SetAzureVMExtensionCommand
```

I have replaced the required variables with my VMs name, location and storage account details as well. I am using DSCForLinux version 1.0 (not sure if this is right though).
Is there any step that I am missing?

**Comment by ahmetalpbalkan** [comment]
@Madhura2016 hi, can you please provide the logs under `/var/log/azure/<dsc-extension-dir>/...` ?

**Comment by Madhura2016** [comment]
@ahmetalpbalkan this is the CommandExecution log 
![image](https://cloud.githubusercontent.com/assets/20465904/16858157/ba7dd47e-49d9-11e6-9f93-892b9cad6d79.png)

**Comment by Madhura2016** [comment]
@ahmetalpbalkan and this is the execution log 
![image](https://cloud.githubusercontent.com/assets/20465904/16858192/00109e9a-49da-11e6-90b5-1bd8b7701c85.png)

**Comment by ahmetalpbalkan** [comment]
@Madhura2016 is it possible for you to send the entire log file as text here?

**Comment by Madhura2016** [comment]
[executionlog.txt](https://github.com/Azure/azure-linux-extensions/files/364927/executionlog.txt)

**Comment by ahmetalpbalkan** [comment]
@lizzha @bingosummer please assist.

**Comment by allanmoller** [comment]
Hey all, aware that this is and old thread, but was this ever resolved, getting exact same error?

**Comment by boumenot** [comment]
No.

There's been zero activity on this, and we are trying to find the (new) owners for DSC.

**Comment by ericgable** [comment]
I can help with this - but I need to see both of the logs below from the Linux VM.
 On my VM the logs for the extension are located here:
```
ls -l /var/log/azure/Microsoft.EnterpriseCloud.Monitoring.OmsAgentForLinux/1.2.75.0/
total 140
-rw-r--r-- 1 root root  11264 Nov 19 18:18 CommandExecution.log
-rw-r--r-- 1 root root 125579 Nov 19 18:18 extension.log

```

**Comment by balukambala** [comment]
Sorry for late reply to this issue. These issues were fixed in the latest version 2.3. Please let us know if you encounter again. Closing this.
